ios/browser/brave_ads/ads_service_impl_ios.h

```diff
@@ -11,6 +11,8 @@
 #include <vector>
```

Comment:
`ShutdownAdsService` or similar should call `is_prefetching_new_tab_page_ad_ = false;`. Please make sure we are copying the logic from `AdsServiceImpl` for these flags.

Reply:
Fixed